Article/Article.js

```diff
@@ -3,18 +3,23 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
+
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerHTML = 'Expand';
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    // console.log('expand');
+    this.domElement.classList.toggle('article-open');
+    this.domElement.classList.toggle('flash');
   }
```

Comment:
remember that syntax i showed you the other day? heres one way you could change the text of the button in this expandArticle function:
```
  this.expandButton.innerHTML = (this.expandButton.innerHTML === 'Expand') ? 'Close' : 'Expand';
```
hopefully this isnt too scary, its just a if-else statement in ES6 syntax
its exactly the same as this 
```
this.expandButton.innerHTML = 
                        if (this.expandButton.innerHTML === 'Expand') { 
                           'Close';
                        }else{
                           'Expand';
                        }